src/components/pages/PageDecksCreate/PageDecksCreate.jsx

```diff
@@ -0,0 +1,252 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { If, Else } from 'react-if';
+import { useSelector } from 'react-redux';
+import MultiSelect from 'react-multi-select-component';
+import { useHistory } from 'react-router';
+import TemplateDefault from '../../Templates/TemplateDefault';
+import firebase from '../../../firebase/firebase';
+import LoadingAnim from '../../atoms/LoadingSpinner/LoadingSpinner';
+
+function PageDecksCreate() {
+  const user = useSelector((state) => state.user);
+  const history = useHistory();
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [characterOptions, setCharacterOptions] = useState([]);
+  const [selectedCharacterIds, setSelectedCharacterIds] = useState([]);
+  const [circumstanceOptions, setCircumstanceOptions] = useState([]);
+  const [selectedCircumstanceIds, setSelectedCircumstanceIds] = useState([]);
+  const [conflictOptions, setConflictOptions] = useState([]);
+  const [selectedConflictIds, setSelectedConflictIds] = useState([]);
+  const [deck, setDeck] = useState({
+    name: '',
+    description: '',
+    visibility: '',
+    characterCards: [{}],
+    circumstanceCards: [{}],
+    conflictCards: [{}],
+  });
+
+  const populateCharacters = useCallback(
+    async () => {
+      try {
+        const cardRefs = await firebase.db
+          .collection('cards')
+          .where('createdBy', '==', user.uid)
+          .where('type', '==', 'Character')
+          .get();
+
+        const newCharacterOptions = cardRefs
+          .docs
+          .map((card) => {
+            const cardData = {
+              id: card.id,
+              ...card.data(),
+            };
+            return { label: cardData.cardTitle, value: cardData.id };
+          });
+
+        setCharacterOptions(newCharacterOptions);
+        setIsLoading(true);
+      } catch (err) {
```

Comment:
If **any** of your populate methods fail you should show an error message to the user.